app/connectors/IdentityVerificationConnector.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.FrontendAppConfig
+import models.iv.{IdentityVerificationEvidenceSource, IdentityVerificationResponse}
+import connectors.IdentityVerificationHttpParser.IdentityVerificationResponseReads
+
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class IdentityVerificationConnector @Inject()(
+                                               httpClient: HttpClient,
```

Comment:
We should use HttpClientV2, for example: https://github.com/hmrc/ioss-returns-frontend/blob/5c8cf741e834e3337960e9cb2522e87676c4eb49/app/connectors/EuVatRateConnector.scala#L31